feature-libs/product-configurator/common/components/service/configurator-router-data.ts

```diff
@@ -7,7 +7,7 @@ export namespace ConfiguratorRouter {
   export interface Data {
     pageType?: PageType;
     isOwnerCartEntry?: boolean;
-    owner?: CommonConfigurator.Owner;
+    owner: CommonConfigurator.Owner;
```

Comment:
This is breaking change.

Reply:
I think @ChristophHi @Platonn agreed on that this is ok. Reason is that the Owner was never really optional. Without the Owner property the configurator is guaranteed not to work. So that this property was optional, can be considered a bug, not a feature.